subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -1,14 +1,14 @@
 #Generated file, please do not edit - Version values used in build-init templates
 commons-math=3.6.1
 groovy=2.5.7
-guava=27.1-jre
+guava=28.0-jre
 junit-jupiter=5.4.2
 junit=4.12
 kotlin=1.3.31
 scala-library=2.12.8
 scala-xml=1.2.0
 scala=2.12
-scalatest=3.0.7
+scalatest=3.0.8
```

Comment:
`updateInitPluginTemplateVersionFile` changed this as well

Reply:
These should be updated automatically during release.

Reply:
k, reverted